# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/render_task_cache.rs
# Commit: 5cee93aa5199
# Full Hash: 5cee93aa5199068b93a13eedd940bf1a3af57011
# Author: Jamie Nicol <jnicol@mozilla.com>
# Date: 2020-07-11 09:22:23
# Regressor Bug: 1645665
# File Overlap Count: 1
# Description:
#   Bug 1645665 - Calculate tight bounds for each sub-run of glyphs when batching. r=lsalzman
#   
#   When adding a text run to a batch, we split the text run in to contiguous
#   runs of glyphs which can be batched together. For example, when a glyph is
#   encountered mid-run with a different glyph format or that is cached in a
# ==============================================================================

diff -r c89ca61cdd43 -r 5cee93aa5199 gfx/wr/webrender/src/render_task_cache.rs
--- a/gfx/wr/webrender/src/render_task_cache.rs	Wed Jul 08 21:45:55 2020 +0000
+++ b/gfx/wr/webrender/src/render_task_cache.rs	Fri Jul 10 18:41:49 2020 +0000
@@ -167,7 +167,7 @@
         // this in the render task. The renderer will draw this
         // task into the appropriate layer and rect of the texture
         // cache on this frame.
-        let (texture_id, texture_layer, uv_rect, _, _) =
+        let (texture_id, texture_layer, uv_rect, _, _, _) =
             texture_cache.get_cache_location(&entry.handle);
 
         render_task.location = RenderTaskLocation::TextureCache {